samples/subsys/logging/syst/sample.yaml

```diff
@@ -20,6 +20,13 @@ tests:
       regex:
         - "SYS-T RAW DATA: "
         - "SYST Sample Execution Completed"
+    # "TOOLCHAIN_HAS_NEWLIB == 1" filter was applied
+    # because of following chain of dependencies:
+    # LOG_MIPI_SYST_ENABLE=y --> CONFIG_MIPI_SYST_LIB --> \
+    # --> REQUIRES_FULL_LIBC --> NEWLIB_LIBC.
+    #
+    # Not all compillers announced in Zephyr support NewLib.
+    filter: TOOLCHAIN_HAS_NEWLIB == 1
```

Comment:
Last minute thought - as I can see 'picolibc' is also treated as a full libc - so probably we should use something like

```
filter: (TOOLCHAIN_HAS_NEWLIB == 1) or CONFIG_PICOLIBC_SUPPORTED
```

In rest - LGTM.

Reply:
Fixed